packages/cli/README.md

```diff
@@ -0,0 +1,50 @@
+# Zowe Native Protocol plug-in for Zowe CLI
+
+The Zowe Native Protocol plug-in for Zowe CLI interacts with the Zowe Native Protocol, a native integration with z/OS systems over SSH, allowing developers to interact with mainframe resources directly from their command line.
+
+## Features
+
+- **Dataset Operations**: Create, read, write, and delete datasets
+- **USS File Access**: Create, read, write, and delete files and directories in Unix System Services
+- **Job Management**: Submit, view, cancel, hold, and release jobs on z/OS
+
+## Minimum Requirements
+
+- Zowe CLI v8.0.0 (or newer)
+
+## Installation
+
+Access the latest version of the CLI plug-in from the GitHub Releases page.
+
+Install the CLI plug-in from the tarball:
+
+```
+zowe plugins install dist/zowe-native-proto-cli-*.tgz
+```
+
+Once complete, the Zowe CLI plug-in is installed and ready to use.
+
+## Usage
+
+Run the `zowe plugins show-first-steps zowe-native-proto-cli` command to see the first steps for using the plug-in.
+
+## Building from source
+
+1. From the root of this repository, run `npm install` to install all the dependencies
+2. Change to the `packages/sdk` folder and run `npm run build` to build the SDK
```

Comment:
This step shouldn't be needed since the CLI depends on the SDK as a TS project reference.

Reply:
Thanks, addressed in upcoming commit